syntax = "proto3";

package grpc.testing;

message Payload {
  bytes body = 2;
}

message SimpleRequest {
  int32 response_size = 2;
  Payload payload = 3;
}

message SimpleResponse {
  Payload payload = 1;
}

service TestService {
  rpc UnaryCall (SimpleRequest) returns (SimpleResponse);
}